VAMobile/src/screens/BenefitsScreen/ClaimsScreen/AppealDetailsScreen/AppealDetailsScreen.tsx

```diff
@@ -110,13 +111,7 @@ const AppealDetailsScreen: FC<AppealDetailsScreenProps> = ({ navigation, route }
           <TextView variant="MobileBody">{t('appealDetails.upToDate', { date: formattedUpdatedDate, time: formattedUpdatedTime })}</TextView>
           <TextView variant="MobileBody">{t('appealDetails.submitted', { date: formattedSubmittedDate })}</TextView>
           <Box mt={theme.dimensions.standardMarginBetween}>
-            <SegmentedControl
-              values={controlValues}
-              titles={controlValues}
-              onChange={setSelectedTab}
-              selected={controlValues.indexOf(selectedTab)}
-              accessibilityHints={segmentedControlA11yHints}
-            />
+            <SegmentedControl labels={controlLabels} onChange={setSelectedTab} selected={controlLabels.indexOf(selectedTab)} labelsA11yHints={segmentedControlA11yHints} />
```

Comment:
Did our variable names change because ya'll made edits to the component itself in the library and we're now matching?

Reply:
Yeah, I combined titles and values props into one since I didn't see the utility in having one array to give the labels shown visually and another array that was only used internally around setting selected state.

That said, there's a chance that'll need to be reworked due to SM. I don't think it's inherently part of the problems I ran into and more on using redux, but also made me realize it may behave finnicky when things change (e.g. the unread count in the segment label). I'll add another note to the separate ticket to be on the lookout for that.